var/spack/repos/builtin/packages/py-astor/package.py

```diff
@@ -17,6 +17,8 @@ class PyAstor(PythonPackage):
     version('0.6', sha256='175ec395cde36aa0178c5a3120d03954c65d1ef4bb19ec4aa30e9d7a7cc426c4')
 
     depends_on('python@2.7:2.8,3.4:')
-    depends_on('py-setuptools', type='build')
+    # Build fails with py-setuptools@41.4.0
+    # https://github.com/berkerpeksag/astor/issues/162
+    depends_on('py-setuptools@:41.0.1', type='build')
```

Comment:
```suggestion
    depends_on('py-setuptools@:43', type='build')
```
In case you know it doesn't build with 43 either

Reply:
Good to know. Since `depends_on('py-setuptools@:41.0.1'...` excludes both v41.4.0 and v43.x.x, I'will keep the commit as it is. Does that make sense?

Reply:
It might be overly strict, but it's likely a hack that we'll remove in the future anyway...